OpenTESArena/src/Game/Game.cpp

```diff
@@ -36,7 +34,8 @@ Game::Game()
 	this->options = OptionsParser::parse();
 
 	// Initialize virtual file system using the Arena path in the options file.
-	VFS::Manager::get().initialize(std::string(this->options->getArenaPath()));
+    std::string arena = File::toString(this->options->getArenaPath());
```

Comment:
It's just a simple static class for common file operations. The only method right now is for reading all of a file's content into a string, but adding a method to check if a file/folder exists would also be good, so I'll do that.